mantle/kola/tests/misc/network.go

```diff
@@ -490,6 +546,20 @@ func checkExpectedMAC(c cluster.TestCluster, m platform.Machine, interfaceName,
 	}
 }
 
+func checkConnectionsUp(c cluster.TestCluster, m platform.Machine, connectionNames []string) error {
+	failedConnections := []string{}
+	for _, connection := range connectionNames {
+		connectionStatus := string(c.MustSSH(m, fmt.Sprintf("nmcli -f GENERAL.STATE connection show %s", connection)))
+		if !strings.HasSuffix(connectionStatus, "activated") {
```

Comment:
`-g` will print just the value so that you don't have to use `HasSuffix`

Reply:
I think I can just use the `--active` flag here..